packages/ember-data/tests/integration/store/query-record-test.js

```diff
@@ -61,3 +65,33 @@ test("When a record is requested, and the promise is rejected, .queryRecord() is
     }));
   });
 });
+
+if (Ember.FEATURES.isEnabled('ds-new-serializer-api')) {
+  test("When a record is requested, the serializer's normalizeQueryRecordResponse method should be called.", function() {
+    expect(1);
+
+    env.registry.register('serializer:person', TestSerializer.extend({
+      normalizeQueryRecordResponse: function(store, primaryModelClass, payload, id, requestType) {
+        equal(payload.data.id , 1, "the normalizeQueryRecordResponse method was called with the right payload");
```

Comment:
I'd add a `return this._super(...arguments);` here just to prevent potential errors/asserts in the future.